anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -56,6 +56,21 @@ let getAnonParams (requestAnonParams: RequestAnonParams) (salt: string) =
     LayerNoiseSD = requestAnonParams.LayerNoiseSD
   }
 
+let prependStarBucketRow suppressedAnonCount result =
+  if suppressedAnonCount <> Null then
+    let starBucketRow =
+      result.Columns
+      |> List.map (
+        function
+        | { Name = "count" } -> suppressedAnonCount
```

Comment:
What if we have a regular column named `count`?

Reply:
hm, true, IIRC `Service` inherits the magic meaning of `count` from `desktop`. But this is a good question to actually check in general, what would happen, I can't remember anymore. As this PR doesn't change this in particular, I'll leave this up to a follow-up (hopefully, without such delay)